bin/oref0-get-ns-entries.js

```diff
@@ -155,15 +155,20 @@ if (!module.parent) {
 
   function loadFromNightscoutWithDate(lastDate, glucosedata) {
 
-    var headers = {
-      'api-secret': apisecret
-    };
+    // append the token secret to the end of the ns url, or add it to the headers if token based authentication is not used
+    var headers = {} ;
+    var tokenAuth = "";
+    if apisecret.startsWith("token=") {
```

Comment:
```suggestion
    if (apisecret.startsWith("token=")) {
```

Passes the checks for me :)

Reply:
Thanks, applied.